x/superfluid/keeper/gov.go

```diff
@@ -1,34 +1,38 @@
 package keeper
 
 import (
+	"fmt"
+
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/osmosis-labs/osmosis/x/superfluid/types"
 )
 
 func (k Keeper) HandleSetSuperfluidAssetsProposal(ctx sdk.Context, p *types.SetSuperfluidAssetsProposal) error {
 	for _, asset := range p.Assets {
 		k.SetSuperfluidAsset(ctx, asset)
-		ctx.EventManager().EmitEvents(sdk.Events{
-			sdk.NewEvent(
-				types.TypeEvtSetSuperfluidAsset,
-				sdk.NewAttribute(types.AttributeDenom, asset.Denom),
-				sdk.NewAttribute(types.AttributeSuperfluidAssetType, asset.AssetType.String()),
-			),
-		})
+		event := sdk.NewEvent(
+			types.TypeEvtSetSuperfluidAsset,
+			sdk.NewAttribute(types.AttributeDenom, asset.Denom),
+			sdk.NewAttribute(types.AttributeSuperfluidAssetType, asset.AssetType.String()),
+		)
+		ctx.EventManager().EmitEvent(event)
```

Comment:
I assume this is just a preferred style thing?

Reply:
Yeah, and I want that event creation to be a different function later.